src/views/DidEndpointsForm/DidEndpointsForm.tsx

```diff
@@ -119,17 +133,44 @@ function DidNewEndpoint({
 
   const dirty = useBooleanState();
 
+  const ns = useAsyncValue(
```

Comment:
Why this cryptic variable name? Maybe this is come convention that I don't know about?

Reply:
This is a standard abbreviation of "namespace", though you’re right, it is not really common anymore. I was going for the shortest name possible in order to make the three following declarations single-line. This didn’t pan out though, so I don’t need this name to be short anymore. Thanks for pointing it out!